tests/Tests.Auth.AwsSigV4/CanonicalRequestTests.cs

```diff
@@ -0,0 +1,123 @@
+/* SPDX-License-Identifier: Apache-2.0
+*
+* The OpenSearch Contributors require contributions made to
+* this file be licensed under the Apache-2.0 license or a
+* compatible open source license.
+*/
+
+namespace Tests.Auth.AwsSigV4
+{
+	using System;
+	using System.Linq;
+	using System.Net.Http;
+	using System.Threading.Tasks;
+	using System.Web;
+	using Amazon.Runtime;
+	using FluentAssertions;
+	using OpenSearch.Net.Auth.AwsSigV4;
+	using OpenSearch.OpenSearch.Xunit.XunitPlumbing;
```

Comment:
Why `using` block is inside `namespace`?

Reply:
Will just be what my IDE was defaulted to due to personal preference, hadn't realised there wasn't a specific setting for this project and that it'd ended up mismatched with the rest of the codebase. Happy to move them outside the namespace if that's desired.

Reply:
I think it is good to move it outside for consistency with the rest project. Thanks!